x-pack/legacy/plugins/ml/public/datavisualizer/datavisualizer_controller.js

```diff
@@ -588,27 +586,25 @@ module
             // TODO - display error in cards saying data could not be loaded.
             console.log('DataVisualizer - error getting non metric field stats from elasticsearch:', err);
             if (err.statusCode === 500) {
-              notify.error(
+              toastNotifications.addDanger(
                 i18n.translate('xpack.ml.datavisualizer.fieldsInternalServerErrorTitle', {
-                  defaultMessage: 'Error loading data for fields in index {index}. {message}. ' +
+                  defaultMessage: 'Error loading data for fields in index {index}. {message}. bob' +
```

Comment:
bob? 👨‍🎨

Reply:
😂 Woops - just a way to differentiate error messages when testing. Seems I left poor Bob in. Will remove.